src/settings.ts

```diff
@@ -18,7 +18,7 @@ import sparkleNavLogo from "assets/icons/sparkle-nav-logo.png";
 import defaultMapIcon from "assets/icons/default-map-icon.png";
 import sparkleverseLogo from "assets/images/sparkleverse-logo.png";
 
-export const SPARKLE_HOMEPAGE_URL = "https://sparklespaces.com/";
+export const SPARKLE_HOMEPAGE_URL = "/v/register/";
```

Comment:
This will actually redirect to a venue page and consider `register` as a venue. Which will fail if it doesn't exist in the ohbm database. 
https://ohbm.sparkle.space/v/register/

There is also being used only in a single place:
```tsx
export const HOMEPAGE_URL = IS_BURN
  ? SPARKLEVERSE_HOMEPAGE_URL
  : SPARKLE_HOMEPAGE_URL;
```
I don't really remember what this `IS_BURN` flag is but make sure you are changing the correct `const`.

Reply:
You can feel free to re-request review if the venue `register` already exists and everything is fine with the redirect.

Reply:
`register` does exist as a venue. This is the change that github asked for.